oracleservice/service_parameters.py

```diff
@@ -3,11 +3,12 @@
 import sys
 import utils
 
+from eth_account.account import Account
```

Comment:
`eth_account` is missing in requirements.txt

Reply:
> `eth_account` is missing in requirements.txt

Added